lib/screens/dungeon/dungeon_list.dart

```diff
@@ -98,48 +101,65 @@ class DungeonListRow extends StatelessWidget {
   Widget build(BuildContext context) {
     var d = _model.dungeon;
     var m = _model.iconMonster;
-    return InkWell(
-      onTap: goToDungeonFn(context, d.dungeonId, null),
-      child: Container(
-          padding: EdgeInsets.symmetric(horizontal: 2, vertical: 4),
-          child: Row(
-            children: <Widget>[
-              PadIcon(d.iconId),
-              SizedBox(width: 8),
-              Expanded(
-                child: Column(
-                  mainAxisAlignment: MainAxisAlignment.start,
-                  crossAxisAlignment: CrossAxisAlignment.start,
-                  children: [
-                    DefaultTextStyle(
-                        style: Theme.of(context).textTheme.caption,
-                        child: Row(children: [
-                          Spacer(),
-                          typeContainer(m?.type1Id, size: 18, leftPadding: 4),
-                          typeContainer(m?.type2Id, size: 18, leftPadding: 4),
-                          typeContainer(m?.type3Id, size: 18, leftPadding: 4),
-                        ])),
-                    FittedBox(alignment: Alignment.centerLeft, child: Text(_model.name())),
-                    DefaultTextStyle(
-                      style: Theme.of(context).textTheme.caption,
-                      child: Row(
-                        mainAxisAlignment: MainAxisAlignment.spaceBetween,
-                        children: [
-                          IconAndText(DadGuideIcons.mp,
-                              _model.maxAvgMp != null ? commaFormat(_model.maxAvgMp) : null),
-                          SizedBox(width: 8),
-                          IconAndText(DadGuideIcons.srank,
-                              _model.maxScore != null ? commaFormat(_model.maxScore) : null),
-                          Spacer(),
-                        ],
-                      ),
-                    ),
-                  ],
-                ),
-              ),
-            ],
-          )),
-    );
+
+    /// Need to use Consumer and thus MultiProvider because the showDungeonMenu function is not a child of the Provider and needs the Provider
```

Comment:
You only need to use /// for top-level comments, it makes them show up in the docs. For comments like this just use //

You also don't really need to add functional comments like this, although I admit I just learned about this solution a few weeks ago, if you look closely you can find other instances in the codebase where I didn't know about this and so I added an unnecessary level of nesting =)

Reply:
Thanks for clearing that up.